warehouse/pom.xml

```
@@ -55,11 +55,12 @@
             <artifactId>spring-boot-configuration-processor</artifactId>
             <optional>true</optional>
         </dependency>
-        <!-- tdEngine -->
+        <!-- https://mvnrepository.com/artifact/com.taosdata.jdbc/taos-jdbcdriver -->
         <dependency>
             <groupId>com.taosdata.jdbc</groupId>
             <artifactId>taos-jdbcdriver</artifactId>
-            <version>2.0.36</version>
+            <version>2.0.42</version>
+        </dependency>
         </dependency>
```

```suggestion
        </dependency>
```
Hi thanks for contribution.  There has double colse tag ` </dependency>`, please remove one to fix it

---

Hi @tomsun28 , Removed the extra dependency tag.